devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -177,6 +180,10 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
     std::mutex m_bufferManagerMutex;
     robometry::BufferManager m_bufferManager;
 
+    const std::string m_rpcPortName{"/YarpRobotLoggerDevice/commands/rpc:i"}; /**< name of Remote
```

Comment:
```suggestion
    const std::string m_rpcPortName{"/commands/rpc:i"}; /**< name of Remote
```

Since you are using the port prefix, maybe we can simplify the rest of the name